src/compiler/crystal/tools/expand.cr

```diff
@@ -21,16 +21,16 @@ module Crystal
         io.puts
         expansion.expanded_sources.zip(expansion.expanded_macros)
                                   .each_with_index do |(expanded_source, expanded_macro), j|
```

Comment:
I should refactor this code in the future, but formatter is not wrong. I believe this indentation is correct and old indent is bad, as @straight-shoota and @Sija said.